assests/javascript/index.js

```diff
@@ -136,6 +151,125 @@ async function updateTaskStatus(id, isDone) {
   }
 }
 
+function handleTitle() {
+  const editTaskIcons = Array.from(document.getElementsByClassName("edit-task-icon"));
+
+  editTaskIcons.forEach((editTaskIcon) => {
+    editTaskIcon.addEventListener("click", (event) => {
+      const id = event.target.parentNode.id;
+      const titleBox = document.getElementById(`title-box-${id}`);
+      const taskTitle = titleBox.innerText;
+
+      convertEditIconToSubmitIcon(id);
+      convertTaskTitleToInput(id, titleBox, taskTitle);
```

Comment:
there is no need to pass `titleBox` as an argument, right? you can just select buy id inside? Or am I missing something?

Reply:
I will check it with you

Reply:
You are right
I just wanted to avoid repeating.